jvm/src/main/scala/PosixLikeIO/PIO.scala

```diff
@@ -13,7 +13,16 @@ import scala.Tuple.Union
 import scala.concurrent.ExecutionContext
 import scala.util.{Failure, Success, Try}
 
+object File:
+  extension (resolver: Future.Resolver[Int])
+    def toCompletionHandler = new CompletionHandler[Integer, ByteBuffer] {
```

Comment:
Not sure if we want this extension method to be public.

Reply:
You are totally right!